src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -203,7 +206,7 @@ const JobHistory = _.flow(
                     status, submissionEntity
                   } = filteredSubmissions[rowIndex]
                   return h(Fragment, [
-                    statusCell(workflowStatuses), status === 'Aborting' && 'Aborting',
+                    statusCell(workflowStatuses), _.keys(collapsedStatuses(workflowStatuses)).length === 1 && status,
```

Comment:
This means you can no longer see an 'Aborting' status if workflows are in multiple states. Are we ok with that?

Reply:
Spoke with Brad about it and a follow up ticket #1511 reflects what future UX could look like!

Reply:
We're okay with this behavior for now